aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -323,7 +330,7 @@ module aptos_framework::account {
         option::fill(&mut account_resource.rotation_capability_offer.for, recipient_address);
     }
 
-    // Accept rotation capability from `offerer_address` if there's an existing rotation capability offer to the account owner in the offerer's account
+    /// Accept rotation capability from `offerer_address` if there's an existing rotation capability offer to the account owner in the offerer's account
     public fun accept_rotation_capability_ed25519(account: &signer, offerer_address: address) : RotationCapability acquires Account {
```

Comment:
On line 340 in this function, you need to wrap the error inside something from `move-stdlib/sources/error.move`, as pointed out before by @movekevin, no?

Can you make sure this holds for all `assert!` calls in this file?